spec/features/users/sign_in_spec.rb

```diff
@@ -318,15 +319,6 @@
       expect(page).to have_field(t('forms.registration.labels.email'), with: '')
       expect(current_url).to match Regexp.escape(sign_up_email_path(request_id: '123abc'))
     end
-
-    it 'does not refresh the page after the session expires', js: true do
-      allow(Devise).to receive(:timeout_in).and_return(60)
-
-      visit root_path
-      expect(page).to_not have_content(
-        t('notices.session_cleared', minutes: IdentityConfig.store.session_timeout_in_minutes),
-      )
-    end
```

Comment:
This test is confusing to me. It was configuring the timeout long enough that I wouldn't expect a refresh regardless of circumstances. And based on the previously-existing logic, the page _does_ refresh even if it follows immediately after the user having been signed out.